scripts/build

```diff
@@ -20,7 +20,7 @@
 cli_build() {
   out="bin"
   CGO_ENABLED=0 go build $GO_BUILD_FLAGS \
-  -o "${out}/swctl" cmd/* || return
+  -o "${out}/swctl" swctl/* || return
```

Comment:
cmd is officially recommended by golang.
e.g.
cmd/swctl/*

Reply:
can you give me some reference?